authority-portal-backend/authority-portal-quarkus/src/main/kotlin/de/sovity/authorityportal/web/UiResourceImpl.kt

```diff
@@ -180,6 +181,7 @@ class UiResourceImpl : UiResource {
     @Transactional
     override fun reactivateParticipantUser(userId: String): IdResponse {
         authUtils.requiresRole(Roles.UserRoles.PARTICIPANT_ADMIN)
+        authUtils.requiresRegistrationStatus(userId, UserRegistrationStatus.DEACTIVATED)
```

Comment:
Not clear who the function refers to, maybe name it `requiresTargetRegistrationStatus` to be more consistent?